src/ocr/ocr.cpp

```diff
@@ -71,6 +81,7 @@ void Ocr::recognize(const QPixmap &pixmap, int dpi)
     m_future = QtConcurrent::run([this, dpi, image = pixmap.toImage()] {
         m_tesseract.SetImage(image.constBits(), image.width(), image.height(), image.depth() / 8, image.bytesPerLine());
         m_tesseract.SetSourceResolution(dpi);
+        m_tesseract.ReadConfigFile(qPrintable(AppSettings().ocrConfigPath()));
```

Comment:
I think that you can just set it directly. BTW, shouldn't we check if the file is empty?
```suggestion
        m_tesseract.ReadConfigFile(AppSettings().ocrConfigPath());
```